hotswap_rgb/fp-lib-table

```diff
@@ -0,0 +1,4 @@
+(fp_lib_table
+  (lib (name "dumbpad")(type "KiCad")(uri "$(KIPRJMOD)/dumbpad.pretty")(options "")(descr ""))
+  (lib (name "Library")(type "KiCad")(uri "D:/Develop/KiCad/dumbpad_v2/Library.pretty")(options "")(descr ""))
```

Comment:
I'm unfamiliar with KiCad - this looks like a local library import. Is this library standard with KiCad or custom? If it's custom, please include it in the PR.

Reply:
here we go, finally ...